index.html

```diff
@@ -49,8 +51,140 @@ <h1 class="boxHead">
+        </section>
+
+        <section class="project">
+            <img src="images/Snapshoot Portfolio (1).png">
+            <p><strong>Multi-Post
+                Stories</strong></p>
+            <p class="tonic">Tonic</p>
+        <div class="list">
+            <p class="canopy">Canopy</p>
+            <ul class="cano">
+                <li>Back End Dev</li>
+                <li>2023</li>
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="lang">
+                <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Java Script</li>
+                
+            </ul>
+            <button class="button">See Project</button>
+        </section>
+
+        <section class="project">
+            <img src="images/Snapshoot Portfolio (2).png">
+            <p><strong>Multi-Post
+                Stories</strong></p>
+                <div class="list">
+                    <p class="canopy">Canopy</p>
+                    <ul class="cano">
+                        <li>Back End Dev</li>
+                        <li>2023</li>
+                    </ul>
+                </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="lang">
+                <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Java Script</li>
+                
+            </ul>
+            <button class="button">See Project</button>
+        </section>
+
+        <section class="project">
+            <img src="images/Snapshoot Portfolio (3).png">
+            <p><strong>Multi-Post
+                Stories</strong></p>
+                <div class="list">
+                    <p class="canopy">Canopy</p>
+                    <ul class="cano">
+                        <li>Back End Dev</li>
+                        <li>2023</li>
+                    </ul>
+                </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="lang">
+                <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Java Script</li>
+                
+            </ul>
+            <button class="button">See Project</button>
+        </section>     
+    </section>  
+
+        <section class="about">
+            <div class="abo">
+            <p class="myself">About Myself</p>
+            <p class="AboutMyself">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <p class="boxConnect">
+                LET’S CONNECT
+            </p>
```

Comment:
- [ ] Please always follows the semantics properly as I mentioned earlier that it makes your website more accessible. Because `Let's CONNECT` is a sub-heading of your `section`, so you need to use the `header` tag instead the `p` tag. Always use the header for the headings of each and every `section` instead of the `p` tag.

**Caution**: *Please use the heading hierarchy properly, for every section there would be a heading and sub-heading but you can't use for example `h2` tag for both*.